packages/yoroi-extension/app/api/ada/lib/storage/bridge/timeUtils.js

```diff
@@ -11,7 +11,7 @@ export type RelativeSlot = {| epoch: number, slot: number |};
  */
 function createEraBoundaries(config: $ReadOnlyArray<CardanoHaskellConfig>): Array<[number, number, CardanoHaskellConfig]> {
   const res: Array<[number, number, CardanoHaskellConfig]> = [];
-  for (let i = 0; i < config.length - 1; i++) {
+  for (let i = 0; i < config.length - 2; i++) {
```

Comment:
I'd rather replace `CoinsPerUtxoWord` with the new `CoinsPerUtxoByte` in the existing networks config than adding a new config with no StartAt and having to do this 🤔 

Or optionally just add `StartAt` into the new config , putting it to the start of Alonzo (if I remember correctly, need to recheck)

Reply:
Anyways, any other change to not have to do this, because this is too easy to forget and create problems in the future

Reply:
The whole BaseConfig thing is not very well designed. There is a lot of unsafe type coercion when working with this. We'll revamp it one day.